### ISSUE TITLE
WCAG 1.3.2 ✅ (Supports): Meaningful Sequence

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5051]
## WCAG 1.3.2: Meaningful Sequence

**Level:** A
**Principle:** Perceivable
**Result:** ✅  Supports (all themes)
**Version** 4bde358b354a2fbdaac0ab4b6a511e4330a79c39

### Description

>When the sequence in which content is presented affects its meaning, a [correct reading sequence](https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence.html#dfn-correct-reading-sequence) can be [programmatically determined](https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence.html#dfn-programmatically-determined).

[WCAG 2.2 Success Criterion 1.3.2](https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence.html)

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [x] Add any new issues to the Accessibility project for triage.
- [x] Replace previous audit in milestone with this update.

### Previous audit

> ✅ Meaningful Sequence
Layout is divided into blocks, with meaning derived from the block grouping rather than the sequence in which the blocks appear on the page, thus there are no concerns about lack of meaningful sequence beyond those already being addressed by structural issues [in 1.3.1]. No work has been identified.

 ### Progress since last audit

There have been no issues identified nor bugs reported. 

 ### Audit Notes

Janeway complies with this criterion primarily through a default of making the DOM order match the visual order([Technique 27]( https://www.w3.org/WAI/WCAG21/Techniques/css/C27)).  There are a few exceptions, for example positioning of article information in a sidebar, where relative position of these blocks is not meaningful.  All of the themes are structured in this way.

 ### Work arising

none
